code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -233,6 +233,7 @@
 	//Runner vars
 	var/savage = FALSE
 	var/savage_used = FALSE
+	var/evasion_stacks = 0
```

Comment:
needs some autodoc commentary describing what this is.

Reply:
Sure